src/object-store/src/layers/lru_cache/read_cache.rs

```diff
@@ -51,23 +51,36 @@ fn can_cache(path: &str) -> bool {
     !path.ends_with("_last_checkpoint")
 }
 
-/// Generate an unique cache key for the read path and range.
-fn read_cache_key(path: &str, range: BytesRange) -> String {
-    format!("{:x}.cache-{}", md5::compute(path), range.to_header())
+/// Generate a unique cache key for the read path and range.
+fn read_cache_key(path: &str, args: &OpRead) -> String {
```

Comment:
So the `range` is removed?

Reply:
I think it's contained within `OpRead`. See `args.range().to_header()` below.